scripts/common/cb_common.sh

```diff
@@ -593,7 +611,14 @@ function mount_filesystem_on_volume {
                 exit 1
             else
                 syslog_netcat "$FILESYS_TYPE filesystem on volume $VOLUME successfully made accessible through mountpoint ${MOUNTPOINT_DIR}"
+                
+                if [[ $(sudo cat /etc/fstab | grep -c ${MOUNTPOINT_DIR}) -eq 0 ]]
+                then
+                    sudo bash -c "echo \"${VOLUME} ${MOUNTPOINT_DIR} ext4  defaults 0 2\" >> /etc/fstab"
```

Comment:
Should `ext4` be hard-coded here? Is it not the same as `$FILESYS_TYPE` ?

Reply:
Of course! My mistake. Fixed and pushed.